specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -1400,6 +1400,539 @@
+        }
+      },
+      "get": {
+        "tags": [
+          "GalleryScripts"
+        ],
+        "operationId": "GalleryScripts_Get",
+        "description": "Retrieves information about a gallery Script Definition.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]+([_]?[a-zA-Z0-9]+)*$",
+            "description": "The name of the Gallery from which the Script Definitions are to be retrieved."
+          },
+          {
+            "name": "galleryScriptName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]+([_]?[a-zA-Z0-9]+)*$",
+            "description": "The name of the gallery Script Definition to be retrieved."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GalleryScript"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a gallery Script.": {
+            "$ref": "./examples/galleryExamples/GalleryScript_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "GalleryScripts"
+        ],
+        "operationId": "GalleryScripts_Delete",
+        "description": "Delete a gallery Script.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]+([_]?[a-zA-Z0-9]+)*$",
+            "description": "The name of the Gallery in which the Script Definition is to be deleted."
+          },
+          {
+            "name": "galleryScriptName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9]+([_]?[a-zA-Z0-9]+)*$",
+            "description": "The name of the gallery Script Definition to be deleted."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          }
```

Comment:
>headers [](http://example.com/codeflow?start=13&length=7)

Could you add Retry-After if you are returning it. 
And better description for each Location and Azure-AsyncOperation ?

Reply:
Added description